lib/slackbotsy/bot.rb

```diff
@@ -95,6 +99,11 @@ def upload(options)
       return nil # be quiet in webhook reply
     end
 
+    ## simple wrapper on api.users (which calls users.list)
+    def users
```

Comment:
@rlister I'm using it, but not in `slackbotsy`. I added it just to expose `api#users` for `Bot` implementations. Is there any other way to expose `api#users`?